### PR TITLE
Conversion of BatchNormalisation cannot deal with list `axis`

### PR DESCRIPTION
**Pull request by fmannhardt**
Fix issue with BatchNormalisation layer having a list `axis` not adding the required Transpose layer when converting a NHWC model. Maybe a regression of the fix here #92

### CODE REVIEW

File: keras2onnx/ke2onnx/batch_norm.py [hunk 1]
```diff
@@ -17,7 +17,12 @@ def convert_keras_batch_normalization(scope, operator, container):
     else:
         raise AttributeError('There is no input_shape or _input_shape for the operator: ' + operator.full_name)
 
-    skip_transpose = (op.axis != shape_len - 1 and op.axis != -1) or shape_len <= 2
+    if (isinstance(op.axis, list)):
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/batch_norm.py]: remove outside bracket

File: keras2onnx/ke2onnx/batch_norm.py [hunk 2]
```diff
@@ -17,7 +17,12 @@ def convert_keras_batch_normalization(scope, operator, container):
...
+        if (len(op.axis) == 1):
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/batch_norm.py]: remove outside bracket

File: keras2onnx/ke2onnx/batch_norm.py [hunk 3]
```diff
@@ -17,7 +17,12 @@ def convert_keras_batch_normalization(scope, operator, container):
...
+            axis = op.axis[0]
+        else: raise AttributeError('No support for more than one axis in: ' + operator.full_name)
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/batch_norm.py]: new line

File: keras2onnx/ke2onnx/batch_norm.py [hunk 4]
```diff
@@ -17,7 +17,12 @@ def convert_keras_batch_normalization(scope, operator, container):
...
+    else: axis = op.axis
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/batch_norm.py]: new line

**Comment by jiafatom** [comment]
Thanks for your PR!

**Comment by fmannhardt** [comment]
You are welcome.

**Comment by wenbingl** [comment]
@fmannhardt , can you response the comments and we can merge your PR soon?

**Comment by fmannhardt** [comment]
I thought since these are all cosmetic changes, you would simply go ahead.

**Comment by vinitra** [comment]
Hello @fmannhardt! It looks like your changes based on @jiafatom's comments haven't persisted to this PR.

**Comment by vinitra** [comment]
Closing this PR as #129 is now merged!
